### PR TITLE
Added user configuration

### PR DESCRIPTION
**Pull request by SilverDragon135**
Added option -u/--user
The newly created service will be running under defined user

+ added annotations about the state of execution of setup.sh
+ added options -q --show-progress for wget (=simplified output)

**I had to replace "=~" with grep. It didn't worked for me on ubuntu 16.04 server edition.** 
(Failed with error `unexpected operator`)

_Tested only on ubuntu 16.04 for now._

### CODE REVIEW

File: masternode/setup.sh [hunk 1]
```diff
@@ -59,16 +85,45 @@ chmod +x geth
...
+     echo "User $USERNAME found."
+  fi
+fi
+
+
 if [ "$SYSTEMD" = true ]; then
     if [ -f /etc/systemd/system/masternode.service ]; then
         sudo systemctl stop masternode && sudo systemctl disable masternode && sudo rm /etc/systemd/system/masternode.service
     fi
+echo '=========================='
```
Comment (iandesj) [inline masternode/setup.sh:107]: This is beautiful. Thanks for this output.

File: masternode/setup.sh [hunk 2]
```diff
@@ -59,16 +85,45 @@ chmod +x geth
...
+echo 'Condiguring service...'
```
Comment (iandesj) [inline masternode/setup.sh]: Typeo: Condiguring -> Configuring

**Review by iandesj — CHANGES_REQUESTED** [review]
Just a typo and we'll proceed. Looks good :+1:

**Comment by SilverDragon135** [comment]
Fixed 😃

**Review by iandesj — APPROVED** [review]

**Pull request merged**